napari/benchmarks/benchmark_text_manager.py

```diff
@@ -51,3 +50,14 @@ def time_add_iteratively(self, n, string):
 
     def time_remove_as_batch(self, n, string):
         self.manager.remove(self.indices_to_remove)
+
+    # `time_remove_as_batch` can only run once per instance;
+    # otherwise it fails because the indices were already removed:
+    #
+    #   IndexError: index 32768 is out of bounds for axis 0 with size 32768
+    #
+    # Why? ASV will run the same function after setup several times in two
+    # occassions: warmup and timing itself. We disable warmup and only
+    # allow one execution per state with these method-specific options:
```

Comment:
If possible, links out to the official ASV docs (e.g. regarding to these parameters) or anything else would be useful here (permalink to ASV code, link to SO).

I guess something similar might be needed for any benchmark that mutates state in an additive/destructive way (e.g. adding or removing elements).